test/integration/multikueue/suite_test.go

```diff
@@ -213,7 +213,7 @@ func managerAndMultiKueueSetup(ctx context.Context, mgr manager.Manager, gcInter
 			Name: "kueue-system",
 		},
 	}
-	gomega.Expect(mgr.GetClient().Create(ctx, managersConfigNamespace)).To(gomega.Succeed())
+	gomega.Expect(client.IgnoreAlreadyExists(mgr.GetClient().Create(ctx, managersConfigNamespace))).To(gomega.Succeed())
```

Comment:
Could we just check if the kueue-system does not found in the cluster before we try to create the namespace?

Reply:
but could we still have a race condition - that we checked and it wasn't there, but it is created a couple of ms later?

Reply:
In this particular test no, the create attempts are done once before the top level specs there is plenty of time in between.

From a generic API POV it could be, and it's a good point for attempting to create it and ignoring the "AlreadyExists" errors.

Reply:
@tenzen-y @mimowo let me know what you think it's the best way to go.

Reply:
out of the two options I prefer ignoring "AlreadyExists", but is there another option: wait in eventually for async creation of kueue-system?

Reply:
The cause it's very simple  `managerAndMultiKueueSetup` it's called twice (currently), depending on the ginkgo parallelism used in the same "cluster" (p=1) or in different "clusters" (p>1).

In case of p=1 we attempt to create it twice in the same cluster.

I personally also prefer the initial , `IgnoreAlreadyExists`, approach.

Reply:
Thanks for explaining, I was also puzzled. In that case we couldn't wait as I suggested before.

Reply:
But could we call managerAndMultiKueueSetup sequentially in this case. I suppose the time gain is negligible here, but it increases complexity

Reply:
> But could we call managerAndMultiKueueSetup sequentially in this case. 

The calls are far apart, One when the suite with GC is starting one when the one without GC starts. 
(For ginkgo parallel runs it happens in different OS processes using different clusters ).

> Is this changed by the recent manager integration reuse PR?

Yes previously the envtest "clusters" wold have bean recreated.

Reply:
Ok another option. Could we create the namespace before calling the function? It is a core namespace for kueue so sounds reasonable to be present before the manager starts

Reply:
That will be somewhere in the framework or extend the framework pass yet another init function.... it's kind of overkill since this is the only place we need it.

Reply:
@trasc @mimowo Thank you for moving this forward. That sounds good to me.